packages/server/index.d.ts

```diff
@@ -0,0 +1,78 @@
+declare module "koa-graphql" {
```

Comment:
I'm adding this to a @types package.

Reply:
on https://github.com/DefinitelyTyped/DefinitelyTyped?

great stuff, tks